src/Pipfile

```diff
@@ -11,4 +11,4 @@ docopt = ">=0.6.2"
 llnl-scraper = {file = "https://github.com/cisagov/scraper/archive/v0.14.0-dev.tar.gz"}
 
 [requires]
-python_full_version = "3.10.10"
+python_full_version = "3.12.3"
```

Comment:
Why is 3.12.3 required for this code?  This doesn't have to be in sync with the Docker container, does it?

Reply:
It was never updated to match the `Dockerfile` due to an oversight. They should always be in sync in this configuration to ensure that the Python package versions pulled by `pipenv` reflect the environment we're deploying them into.

Reply:
Also this oversight is why `pipenv` installation will be done [with the following](https://github.com/cisagov/skeleton-docker/blob/ae46c280e3af456b59c271c85c95719180cb6da2/Dockerfile#L51-L52) upstream (and filter down to this project):

```dockerfile
RUN pipenv check --verbose \
    && pipenv install --clear --deploy --extra-pip-args "--no-cache-dir" --verbose
```

This _should_ fail if the versions do not match.